doc/source/getting_started/write_doc/index.rst

```diff
@@ -0,0 +1,167 @@
+.. _ref_write_doc:
+
+=============
+Documentation
+=============
+
+Writing good documentation for a GitHub repository is crucial to ensure that
```

Comment:
Again, this is defined in the PyAnsys developer documentation (see [here](https://dev.docs.pyansys.com/doc-style/index.html))